omp/test/matrix/csr_kernels.cpp

```diff
@@ -776,7 +776,6 @@ TEST_F(Csr, ComputeSubmatrixFromIndexSetIsEquivalentToRef)
         this->ref, row_nnz.get_data(), row_nnz.get_num_elems());
     auto num_nnz = row_nnz.get_data()[rset.get_num_elems()];
     auto drow_nnz = gko::Array<int>(this->omp, row_nnz);
-    drow_nnz.fill(gko::one<int>());
```

Comment:
does it give error?

Reply:
No, I was doing it incorrectly. I am already copying from row_nnz, so there is no need to fill it after that.